apps/dashboard/src/app/globals.css

```diff
@@ -20,6 +20,10 @@
   }
 }
 
+input[type=number] {
```

Comment:
We'll also need the following to remove the arrows on Chrome:

<img width="311" alt="Screenshot 2023-12-29 at 10 04 25" src="https://github.com/QuiiBz/ogstudio/assets/43268759/935df353-89fb-415f-8a23-af01bcebdc27">

```css
input[type=number]::-webkit-outer-spin-button, input[type=number]::-webkit-inner-spin-button {
    -webkit-appearance: none;
}
```